### PR TITLE
[16.0] [MIG] purchase_request

### PR DESCRIPTION
**Pull request by Nikul-OSI**
@dreispt Here need to check **analytic_tag_ids & analytic_account_id** code & functionality.

### CODE REVIEW

**Comment by dreispt** [comment]
@diogocsc please review

File: purchase_request/views/purchase_request_line_view.xml [hunk 1]
```diff
@@ -585,11 +585,11 @@
                                 groups="analytic.group_analytic_accounting"
                                 attrs="{'readonly': [('is_editable','=', False)]}"
                             />
-                        <field
+                        <!-- <field
                                 name="analytic_tag_ids"
                                 groups="analytic.group_analytic_accounting"
                                 attrs="{'readonly': [('is_editable','=', False)]}"
-                            />
+                            /> -->
```
Comment (dreispt) [inline purchase_request/views/purchase_request_line_view.xml:592]: Should this be replaces by the Analytic Distribution new field?

File: purchase_request/views/purchase_request_view.xml [hunk 2]
```diff
@@ -174,10 +177,10 @@
                                         name="analytic_account_id"
                                         groups="analytic.group_analytic_accounting"
                                     />
-                                    <field
+                                    <!-- <field
                                         name="analytic_tag_ids"
                                         groups="analytic.group_analytic_accounting"
-                                    />
+                                    /> -->
```
Comment (dreispt) [inline purchase_request/views/purchase_request_view.xml:183]: Analytic Distribution?

**Review by dreispt — COMMENTED** [review]
I think the new Analytic Distribution, replacing Analytic Tags, needs to be addressed.

**Comment by StefanRijnhart** [comment]
I filed an attempt simultaneously here: https://github.com/OCA/purchase-workflow/pull/1653

**Comment by dreispt** [comment]
@Nikul-OSI Can you please review Stefan's migration and compare to yours? A decision will be needed on which one to move ahead.

**Comment by Nikul-OSI** [comment]
> @Nikul-OSI Can you please review Stefan's migration and compare to yours? A decision will be needed on which one to move ahead.

Hello @dreispt, In @StefanRijnhart PR added a **Analytic Distribution** related code, so better to close this and #1592 PR's.

we are move ahead with https://github.com/OCA/purchase-workflow/pull/1653

Thank you.

**Comment by StefanRijnhart** [comment]
Thanks @Nikul-OSI! Closing as per your comment.

**Pull request closed without merging**